.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Fixes #
+
+### Proposed fixes:
+
+
+
+### Features:
+
+- [ ] includes tests covering changes
+- [ ] includes updated documentation
+- [ ] includes user-visible changes
+- [ ] includes API changes
+- [ ] includes bugfix for possible backport
```

Comment:
"please [X] all the boxes that apply" just to make it clear we're not requiring all of the above